_posts/2020-09-11-liver-patient-classification/2020-09-11-liver-patient-classification.Rmd

```diff
@@ -0,0 +1,717 @@
+It contains data for 583 patients collected in the north east of Andhra Pradesh, one of the 28 states of India.
+The observations are divided into two classes based on the patient having liver disease or not.
+Besides the class variable, which is our target, ten, mostly numerical, features are provided.
+To describe the features in more detail, the table below lists the variables included in the dataset.
+
+```{r, echo=FALSE}
+col_descr <- cbind.data.frame(Variable = colnames(liver_df),
+                              Description = c("Age of the patient (all patients above 89 are labelled as 90",
+                                              "Sex of the patient (1 = female, 0 = male)",
+                                              "Total serum bilirubin (in mg/dL)",
+                                              "Direct bilirubin level (in mg/dL)",
+                                              "Serum alkaline phosphatase level (in U/L)",
+                                              "Serum alanine transaminase level (in U/L)",
+                                              "Serum aspartate transaminase level (in U/L)",
+                                              "Total serum protein (in g/dL)",
+                                              "Serum albumin level (in g/dL)",
+                                              "Albumin-to-globulin ratio",
+                                              "Target variable (1 = liver disease, 0 = no liver disease)"
+                                              )
+                              )
+
+kable(col_descr)
+```
+
+As one can see, besides age and gender, the dataset contains eight additional numerical features.
+While the names and corresponding measurements look rather cryptic to the uninformed eye, they are all part of standard blood tests conducted to gather information about the state of a patient's liver, so-called liver function tests.
+All of these measurements are frequently used markers for liver disease.
+For the first five, measuring the chemical compound bilirubin and the three enzymes alkaline phosphatase, alanine transaminase and aspartate transaminase, elevated levels indicate liver disease @Gowda2009 @Oh2011.
+For the remaining three, which concern protein levels, lower-than-normal values suggest a liver problem @Carvalho2018 @LabTests2019.
+Lastly, one should note that some of the measurements are part of more than one variable. For example, the total serum bilirubin is simply the sum of both the direct and indirect bilirubin levels and the amount of albumin is used to calculate the values of the total serum protein as well as the albumin-to-globulin ratio. So, one might already suspect that some of the features are highly correlated to one another, but more on that kind of analysis in the following section. 
+
+# Data exporation
+
```

Comment:
Write one sentence for each plot describing what we should see there or drop the plot.